upgrade_tests/test_existance_relations/cli/test_subscriptions.py

```diff
@@ -25,7 +25,7 @@
 # Required Data
 component = 'subscription'
 sub_name = compare_postupgrade(component, 'name')
-sub_uuid = compare_postupgrade(component, ('id', 'uuid', 'uuid', 'uuid'))
+sub_uuid = compare_postupgrade(component, ('id', 'uuid', 'uuid', 'uuid', 'uuid'))
```

Comment:
@jyejare why we are cherry-picking and adding data from 6.5 in 6.3.z branch

Reply:
@ntkathole , The current implementation in existence Tests mandates for keeping the data for all versions regardless of branch/sat_version.

The plan is to keep only TO_VERSION and FROM_VERSION variants in TO_VERSION branch.

So when I will raise that PR, I will be able to easily cherry-pick in all the branches as the implementation in all branches will be same(it is just to avoid/lessen the conflicts in cherry-pick basically).